docs/releases/v1.23.2.md

```diff
@@ -0,0 +1,93 @@
+# Kubernetes Fury Distribution Release v1.23.1
+
+Welcome to the KFD release `v1.23.1`. This is a patch release
+fixing bugs in all the core modules.
+
+The team has been working to make the release upgrade as simple as possible, so read carefully the upgrade path of each
+core module listed below along with the upgrade path of the distribution.
+
+⚠️ If upgrading from `v1.23.0`, you **must** delete all the objects (StatefulSet, Deployment, DaemonSet, etc) as specified in the release notes of the modules
+**before** upgrading to `v1.23.1`.
+
+This distribution is maintained with ❤️ by the team [SIGHUP](https://sighup.io/),
+and is battle tested in production environments.
+
+## New Features
+
+### Core Module Updates
+
+- Removed `commonLabels` from all the `kustomize` katalogs
+
+- [networking](https://github.com/sighupio/fury-kubernetes-networking) 📦 core module: v1.8.2 -> [**v1.9.0**](https://github.com/sighupio/fury-kubernetes-networking/releases/tag/v1.9.0)
+  - Update [Calico] from version `3.21.3` to `3.23.2`.
+- [monitoring](https://github.com/sighupio/fury-kubernetes-monitoring) 📦 core module: v1.14.1 -> [**v1.14.2**](https://github.com/sighupio/fury-kubernetes-monitoring/releases/tag/v1.14.2)
+  - Bugfix on [x509-exporter] 
+- [logging](https://github.com/sighupio/fury-kubernetes-logging) 📦 core module: v1.10.2 -> [**v1.10.3**](https://github.com/sighupio/fury-kubernetes-logging/releases/tag/v1.10.3)
+  - Update [fluent-bit] from version `1.8.10` to `1.9.5`.
+- [ingress](https://github.com/sighupio/fury-kubernetes-ingress) 📦 core module: v1.12.2 -> [**v1.12.2**](https://github.com/sighupio/fury-kubernetes-ingress/releases/tag/v1.12.2)
+  - No updates
+- [dr](https://github.com/sighupio/fury-kubernetes-dr) 📦 core module: v1.9.2 -> [**v1.9.2**](https://github.com/sighupio/fury-kubernetes-dr/releases/tag/v1.9.2)
+  - No updates
+- [OPA](https://github.com/sighupio/fury-kubernetes-opa) 📦 core module: v1.6.2 -> [**v1.6.2**](https://github.com/sighupio/fury-kubernetes-opa/releases/tag/v1.6.2)
+  - No updates
+
+> Please refer the individual release notes for detailed information
+
+## Upgrade path
+
+### Katalog Procedure
+To upgrade the distribution from `v1.23.1` to `v1.23.2` please follow the instructions written in the release notes of each core module.
```

Comment:
I don't think there are specific steps to perform, there are no breaking changes nor manual tasks do to AFAIK to upgrade from `v1.23.1` to `v1.23.2`.

Am I missing something?

Reply:
I removed the sentence